<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>402</x>
        <y>140</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>402</x>
        <y>220</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_signin">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign-in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_signup">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign-up</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineEdit_id_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_passconfirm">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nickname">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>310</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_confirm">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>400</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>400</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_check_id">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>duplication check</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_check_nick">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>310</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>duplication check</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>confirm password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>290</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>nickname</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>321</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Password's length must be between 6 and 10 </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_logout">
      <property name="geometry">
       <rect>
        <x>768</x>
        <y>40</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>logout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_entrance">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>450</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>entrance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_makeroom">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>450</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>make room</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>current chatting room</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_room">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>110</y>
        <width>211</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_welcome">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QListWidget" name="listWidget_users">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>110</y>
        <width>211</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_invite">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>450</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>invite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>connected users</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_welcome_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_logout_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>40</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>logout</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_message">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>409</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_send">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>410</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_attachment">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>450</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>attachment</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_chat">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>80</y>
        <width>591</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_exit">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>received file</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_roomname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>341</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
